openapi.yaml

```diff
@@ -70,7 +70,7 @@ paths:
           $ref: '#/components/responses/NotFound'
     get:
       summary: List special events
-      description: Return a list of upcoming special events at the museum.
+      description: Return a list of past special events at the museum. See one you like? Use this API to [buy a ticket](/#tag/Tickets/operation/buyMuseumTickets).
```

Comment:
I think this should still be upcoming special events rather than past ones - we may need to add a date filter and/or some further-in-the-future events!

And the URL should be a full URL, so that the description markdown is useful in any context.